063-moving-data-between-two-jbod-disks-using-cruise-control.md

```diff
@@ -0,0 +1,71 @@
+# Moving data between two JBOD disks using Cruise Control
+
+This proposal is about integrating the `remove_disks` endpoint from Cruise Control into Strimzi cluster operator. 
```

Comment:
I can't find any documentation about this endpoint in Cruise Control, for example it's missing in the wiki.
Is there an official place where we can find how it works and put a link here?

Reply:
Do we want to link to the [yaml](https://github.com/ilievladiulian/cruise-control/blob/ad275079f271b28704a1d5ff00fff82891aebc89/cruise-control/src/main/resources/yaml/endpoints/removeDisks.yaml)

Reply:
I will create a documentation bug and create a PR for the same on upsteam repo regarding this. The only piece of information for using this is only on the PR